docker-compose.imgproxy.yaml

```diff
@@ -2,18 +2,22 @@
 services:
     imgproxy:
         container_name: ddev-${DDEV_SITENAME}-imgproxy
-        hostname: ${DDEV_SITENAME}-imgproxy
         image: darthsim/imgproxy:latest
+        restart: "no"
+        # These labels ensure this service is discoverable by ddev.
         labels:
             com.ddev.site-name: ${DDEV_SITENAME}
             com.ddev.approot: $DDEV_APPROOT
         networks: [default, ddev_default]
         environment:
+            VIRTUAL_HOST: $DDEV_HOSTNAME
+            HTTPS_EXPOSE: 8081:8080
+            HTTP_EXPOSE: 8080:8080
             IMGPROXY_BASE_URL: http://web:80
```

Comment:
This should be `http://web:80/`

I tried to open images in the webroot (not in subfolders), and it didn't work until I change the env.